types/shared/Entity.ts

```diff
@@ -0,0 +1,16 @@
+import {URL} from './utils';
```

Comment:
Also solved the duplicate `Entity` definition error via #289 - sorry for conflicts here!